go/src/net/Session.go

```diff
@@ -0,0 +1,117 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package nebulaNet
+
+import (
+	"fmt"
+
+	graph "github.com/vesoft-inc/nebula-clients/go/nebula/graph"
+)
+
+type Session struct {
+	sessionID  int64
+	connection *Connection
+	connPool   *ConnectionPool
+}
+
+func newSession(sessionID int64, connection *Connection,
+	connPool *ConnectionPool) Session {
+	newObj := Session{}
+	newObj.sessionID = sessionID
+	newObj.connection = connection
+	newObj.connPool = connPool
+
+	return newObj
+}
+
+// unsupported
+// func (session *Session) ExecuteJson(stmt string) (*graph.ExecutionResponse, error) {
+// 	return session.graph.ExecuteJson(session.sessionID, []byte(stmt))
+// }
+
+func (session *Session) Execute(stmt string) (*graph.ExecutionResponse, error) {
+	resp, err := session.connection.Execute(session.sessionID, stmt)
+	if err != nil {
+		_err := session.reConnect()
```

Comment:
please check the error type then judge if need reconnect, the err may many kinds, such as timeout

Reply:
Added check for EOF, will add timeout soon